python/taichi/lang/exception.py

```diff
@@ -4,3 +4,7 @@ class TaichiSyntaxError(Exception):
 
 class InvalidOperationError(Exception):
     pass
+
+
+class TaichiError(Exception):
```

Comment:
What about renaming this to `TaichiCompilationError`? `TaichiError` sounds a bit weird.